include/vrb/Texture.h

```diff
@@ -17,10 +17,12 @@ namespace vrb {
 class Texture {
 public:
   static TexturePtr Create(ContextWeak& aContext);
+  static TexturePtr Create(ContextWeak& aContext, GLenum aTarget);
```

Comment:
It's not clear to me that this function is needed?

Reply:
Function deleted. I's not used.